lagom-cloud-object-storage-example/account-impl/src/main/java/com/lightbend/lagom/account/impl/readside/ReportWriter.java

```diff
@@ -0,0 +1,32 @@
+package com.lightbend.lagom.account.impl.readside;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class ReportWriter {
+  static String write(Report report) {
+
+    StringBuffer sb = new StringBuffer();
+
+    sb.append("ACCOUNT: ").append(report.accountNumber).append("\n");
+    sb.append("REPORT: ").append(report.getId()).append("\n");
+
+    sb.append("---------------------------------------------------------\n");
+    sb.append("START BALANCE\t\t\t\t\t").append(report.startBalance).append("\n");
+
+    for (Transaction tx: report.getTransactions()) {
+      sb.append(StringUtils.rightPad(tx.getLabel(), 8))
+        .append("\t")
+        .append(tx.getDateTime())
+        .append("\t")
+        .append(tx.getAmount())
+        .append("\n");
+
+    }
+
+    sb.append("END BALANCE\t\t\t\t\t").append(report.endBalance).append("\n");
+    sb.append("---------------------------------------------------------\n");
+
+    return sb.toString();
```

Comment:
I think `StringBuilder` is recommended over `StringBuffer` when thread-safety is not required. 

Also, would it make sense to use Akka's `ByteString` instead?

Reply:
I wanted it also for logging. It that a real concern in using `StringBuffer`?

Reply:
We can keep `StringBuffer` and add a comment: "This string buffer simulates a slow report generation".